bootstrap.sh

```diff
@@ -68,6 +68,11 @@ source utils.sh
 
 #----
 
+# add name to sudoers
+name=$(whoami)
+sudo echo '$(name)	ALL=(ALL) NOPASSWD: ALL' >> /etc/sudoers
```

Comment:
this is wrong, right? did you test it? needs to be " instead of ' and ${name} instead of $(name).

Reply:
Fixed.